cognite_toolkit/_repo_files/AzureDevOps/.devops/dry-run-pipeline.yml

```diff
@@ -0,0 +1,33 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+      - '*'
+
+jobs:
+- job: DeployModules
+  displayName: 'Dry-run test in Dev'
+  pool:
+    vmImage: 'ubuntu-latest'
+  container:
+    image: cognite/toolkit:0.3.1
+  variables: 
+    - group: 'dev-toolkit-credentials'
+  steps:
+    - checkout: self
+      clean: true
+    
+    - script: |
+        cdf build
+      displayName: 'Build the modules'
```

Comment:
Same here

Reply:
no way to do it cleaner, so I just duplicated them